alphagov/paas-admin
# Introduce jest

What
----

We're refactoring the tests to use `jest` instead of types.

It brings many benefits, among which we:
- Could get rid of nock <3
- Can focus tests programatically
- Run tests faster
- Make them prettier
- Make them easier to understand for Ginkgo/rspec comers

**This PR also upgrades the govuk-frontend!**

We were running out of date, but also we could get rid of the legacy `govuk_template_jinja` module. This also means we can remove some of the hack-arounds we've provided for `webpack`.

How to review
-------------

- Code review
- Run application
- Navigate through and see if nothing is out of place
- Run tests
